Documentation/UsingSetting/UserTSconfig.rst

```diff
@@ -0,0 +1,206 @@
+however should **not** be changed or set directly without using the API.
+
+
+.. index:: pair: User TSconfig; Verify configuration
+.. _userverifyingthefinalconfiguration:
+
+Verify the final configuration
+==============================
+
+The full User TSconfig of the currently logged in backend user can be viewed
+using the :guilabel:`System > Configuration` module and choosing the
+action :guilabel:`$GLOBALS['BE_USER']->getTSConfig() (User TSconfig)`. However
+this module can only be accessed by admins.
+
+.. figure:: /Images/ManualScreenshots/Configuration/UserTSconfigOverview.png
+    :alt: Viewing User TSconfig using the Configuration module
+
+
+.. index:: pair: User TSconfig; Override values
+.. _user-override-modify-values:
+
+Override and modify values
+===========================
+
+Properties, which are set in the TSconfig field of a group, are valid
+for all users of that group.
+
+Values, which are set in one group, can be overridden and
+:ref:`modified <t3coreapi:typoscript-syntax-syntax-value-modification>` in the same or
+another group. If a user is member of multiple groups, the TSconfig
+settings are evaluated in *the* order, in which the groups are included
+in the user account: When you are editing the backend user, the
+selected groups are evaluated from top to bottom.
+
+**Example:**
+
+* Add in User TSconfig
+
+.. code-block:: typoscript
+
+	page.RTE.default.showButtons = bold
+
+* You get the value "bold".
+
+* Add later in User TSconfig
+
+.. code-block:: typoscript
+
+	page.RTE.default.showButtons := addToList(italic)
+
+* You get the value "bold,italic".
+
+Finally you can override or
+:ref:`modify <t3coreapi:typoscript-syntax-syntax-value-modification>`
+settings from groups, of which your user is a member, in the User TSconfig
+field of that user himself.
+
+**Example:**
+
+Let's say the user is a member of a *usergroup* with this
+configuration
+
+.. code-block:: typoscript
+
+	TCAdefaults.tt_content {
+		hidden = 1
+		header = Hello!
+	}
+
+Then we set the following values in the TSconfig field of the *user*
+himself
+
+.. code-block:: typoscript
+
+	TCAdefaults.tt_content.header = 234
+	options.clearCache.all = 1
+
+This would override the default value of the header ("234") and add the
+clear cache option. The default value of the hidden field is not
+changed and simply inherited directly from the group.
+
+
+.. index:: User TSconfig; Override page TSconfig
+.. _userrelationshiptovaluessetinpagetsconfig:
+.. _pageoverridingpagetsconfigwithusertsconfig:
+
+Overriding Page TSconfig in User TSconfig
+=========================================
+
+All properties from Page TSconfig can be **overridden** in User TSconfig by
+prepending the property name with `page.`. When a Page TSconfig property is
+set in **User** TSconfig that way, no matter, if in the TSconfig field of a
+group or a user, it **overrides** the value of the according **Page**
+TSconfig property.
```

Comment:
I don't understand this part. Does the "no matter" mean "it has no impact"?

Reply:
user tsconfig always overrides page tsconfig. this applies to user tsconfig set in the user and user tsconfig set in the users group... how do we we formulate this better?

Reply:
Oh! Okay.  How about this:
"When a Page TSconfig property is set in **User** TSconfig that way, regardless of whether it is in the TSconfig field of a
group or a user, it **overrides** the value of the according **Page** TSconfig property."